django/db/backends/sqlite3/creation.py

```diff
@@ -52,7 +54,10 @@ def get_test_db_clone_settings(self, suffix):
         orig_settings_dict = self.connection.settings_dict
         source_database_name = orig_settings_dict['NAME']
         if self.is_in_memory_db(source_database_name):
-            return orig_settings_dict
+            if multiprocessing.get_start_method() == 'spawn':
+                return {**orig_settings_dict, 'NAME': f'{self.connection.alias}_{suffix}.sqlite3'}
+            elif multiprocessing.get_start_method() == 'fork':
+                return orig_settings_dict
```

Comment:
As the start method can also be `forkserver` we ought to guard against that and any potential future values instead of falling through to the implicit `return None`.

In addition, we can remove some indentation here:

```python
    def get_test_db_clone_settings(self, suffix):
        settings_dict = self.connection.settings_dict

        source_database_name = settings_dict['NAME']
        if not self.is_in_memory_db(source_database_name):
            root, ext = os.path.splitext(source_database_name)
            return {**settings_dict, 'NAME': f'{root}_{suffix}{ext}'

        start_method = multiprocessing.get_start_method()
        if start_method == 'fork':
            return settings_dict
        if start_method == 'spawn':
            return {**settings_dict, 'NAME': f'{self.connection.alias}_{suffix}.sqlite3'}
        raise NotImplementedError(f'Cloning with start method {start_method!r} is not supported.')
```

Don't forget to add a test for the exception.